main.tf

```diff
@@ -31,4 +31,14 @@ resource "google_redis_instance" "default" {
   reserved_ip_range = "${var.reserved_ip_range}"
 
   labels = "${var.labels}"
+
+  depends_on = ["google_project_service.redis"]
+}
+
+resource "google_project_service" "redis" {
+
```

Comment:
Heads up - `terraform fmt` will remove this space, so we might as well remove it in the PR.